man/man8/zfs-scrub.8

```diff
@@ -0,0 +1,50 @@
+.\"
+.\" CDDL HEADER START
+.\"
+.\" The contents of this file are subject to the terms of the
+.\" Common Development and Distribution License (the "License").
+.\" You may not use this file except in compliance with the License.
+.\"
+.\" You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+.\" or https://opensource.org/licenses/CDDL-1.0.
+.\" See the License for the specific language governing permissions
+.\" and limitations under the License.
+.\"
+.\" When distributing Covered Code, include this CDDL HEADER in each
+.\" file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+.\" If applicable, add the following below this CDDL HEADER, with the
+.\" fields enclosed by brackets "[]" replaced with your own identifying
+.\" information: Portions Copyright [yyyy] [name of copyright owner]
+.\"
+.\" CDDL HEADER END
+.\"
+.\" Copyright 2024 Klara, Inc.
+.\" Copyright 2024 Mariusz Zaborski <oshogbo@FreeBSD.org>
+.\"
+.Dd February 21, 2024
+.Dt ZFS-SCRUB 8
+.Os
+.
+.Sh NAME
+.Nm zfs-scrub
+.Nd run scrub of file in ZFS storage pools
+.Sh SYNOPSIS
+.Nm zfs
+.Cm scrub
+.Ar filename Ns
+.
+.Sh DESCRIPTION
+Runs a scrub for a single file.
+The scrub examines all data associated with a given file.
+For replicated
+.Pq mirror, raidz, or draid
+devices, ZFS automatically repairs any damage discovered during the scrub.
+The
+.Nm zpool Cm status
+command reports the progress of the scrub and summarizes the results of the
+scrub upon completion.
+For more details about scrubing, refer
+.Xr zpool 8 .
+.
+.Sh SEE ALSO
+.Xr zpool-status 8
```

Comment:
Maybe also reference zpool-scub here:
```
.Sh SEE ALSO
.Xr zpool-status 8 ,
.Xr zpool-scrub
```
And then in `zpool-scrub.8` reference zfs-scrub:
```
.Sh SEE ALSO
.Xr systemd.timer 5 ,
.Xr zpool-iostat 8 ,
.Xr zpool-resilver 8 ,
.Xr zpool-status 8 ,
.Xr zfs-scrub 8
```

Reply:
Will do